app/assets/stylesheets/components/_option-select.scss

```diff
@@ -103,4 +103,44 @@
       }
     }
   }
+
+  // This is a temporary overwrite for checkboxes
+  // These styles should be removed in favour of small checkboxes from the Design System
+  .govuk-checkboxes__item {
+    min-height: 26px;
```

Comment:
This `min-height` seems to be overriding the larger min-height already defined, but does it need to be specified as 26px? Seems to work okay if it's reset, i.e. set to 0, but there may be a browser/usage case I'm unaware of. Just curious.

Reply:
Good question. Maybe not, but I needed to overwrite the existing one anyway.